spec/services/online_codes_import_service_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe OnlineCodesImportService do
+  context 'when importing online codes from csv file' do
+    it 'imports all online codes from csv file' do
+      create(:reward, title: 'RewardTest1')
+      create(:reward, title: 'RewardTest2')
+      params = { file: fixture_file_upload('./spec/fixtures/online_codes/valid_online_codes_import.csv') }
+      import_online_codes = described_class.new(params)
```

Comment:
Hint: if you call such a variable `service`, it is always obvious what it is. `import_online_codes` doesn't match service's class name, so it may be not readable at first glance.

Reply:
Understood! I have changed it :) Thanks!